### PR TITLE
Implement libjn:remove-keys

### PR DESCRIPTION
**Pull request by riyafa**

### CODE REVIEW

File: vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/json/LibjnRemoveKeysScalarEvaluator.java [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        abvs = new ArrayBackedValueStorage();
+        abvs1 = new ArrayBackedValueStorage();
+        sb = new SequenceBuilder();
+        op = (ObjectPointable) ObjectPointable.FACTORY.createPointable();
+    }
+
+    @Override
+    protected void evaluate(TaggedValuePointable[] args, IPointable result) throws SystemException {
+        TaggedValuePointable sequence = args[0];
+        TaggedValuePointable keys = args[1];
```
Comment (prestoncarman) [inline vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/json/LibjnRemoveKeysScalarEvaluator.java:60]: Type check the values before operating on the values. (Basically have an if statement to confirm type and throws an error if the type is incorrect.

File: vxquery-xtest/src/test/resources/cat/LibrariesInJSONiq.xml [hunk 2]
```diff
@@ -40,4 +40,9 @@
       <query name="flatten" date="2016-07-20"/>
       <output-file compare="Text">flatten.txt</output-file>
    </test-case>
+    <test-case name="remove-keys" FilePath="Json/Libraries/" Creator="Riyafa Abdul Hameed">
```
Comment (prestoncarman) [inline vxquery-xtest/src/test/resources/cat/LibrariesInJSONiq.xml:43]: Please add two more tests: use a string and a empty sequence for the keys value..

**Comment by prestoncarman** [comment]
Please squash and prepare for commit.

**Comment by prestoncarman** [comment]
I think this PR has been wrapped up in another PR. Please close.
